raiden-dapp/vue.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 module.exports = {
+  publicPath: process.env.DEPLOYMENT === 'staging' ? '/staging/' : '/',
```

Comment:
Don't we need to exclude `/staging` from the SPA router as well?

Reply:
I have no idea let me check